### PR TITLE
[Feature] Add internal usePackagingFeatureConfig flag

### PR DESCRIPTION
**Pull request by KaterinaWire**
### Issues
We need to cover the upcoming packaging feature with an internal flag so that we can silently release it next time.

### Solutions
Add the `usePackagingFeatureConfig` property to the `WireCallCenterV3` class and set it from the UI project.

### CODE REVIEW

**Review by johnxnguyen — APPROVED** [review]

**Comment by johnxnguyen** [comment]
You might need to put some of the new tests (for starting calls) behind this flag too.

**Comment by KaterinaWire** [comment]
> You might need to put some of the new tests (for starting calls) behind this flag too.

I added a test

File: Tests/Source/Calling/WireCallCenterV3Tests.swift [hunk 1]
```diff
@@ -648,6 +649,20 @@ class WireCallCenterV3Tests: MessagingTest {
         // then
         XCTAssertNil(mockAVSWrapper.startCallArguments)
     }
+
+    func testThatItStartsAConferenceCall_IfConferenceCallingFeatureIsDisabledByInternalFlag(){
```
Comment (David-Henner) [inline Tests/Source/Calling/WireCallCenterV3Tests.swift:653]: The name is a bit confusing to me, it sounds like the conference calling feature is disabled but we should still be able to start a conference call? 
Should it be something like `testThatItStartsAConferenceCall_IfPackagingFeatureIsDisabledByInternalFlag()` instead?

Comment (KaterinaWire) [reply Tests/Source/Calling/WireCallCenterV3Tests.swift]: Yes, fixed

**Review by David-Henner — APPROVED** [review]

**Pull request merged**